src/CMakeLists.txt

```diff
@@ -75,6 +75,13 @@ if (MACOS_BUNDLE)
 	set(RESOURCE_FILES "${CMAKE_SOURCE_DIR}/src/resource/cemu.icns")
 	target_sources(CemuBin PRIVATE "${RESOURCE_FILES}")
 
+	set(MACOSX_BUNDLE_ICON_FILE "cemu.icns")
+	set(MACOSX_BUNDLE_GUI_IDENTIFIER "info.cemu.Cemu")
+	set(MACOSX_BUNDLE_BUNDLE_NAME "Cemu")
+	set(MACOSX_BUNDLE_SHORT_VERSION_STRING ${CMAKE_PROJECT_VERSION})
+	set(MACOSX_BUNDLE_BUNDLE_VERSION ${CMAKE_PROJECT_VERSION})
+	string(TIMESTAMP MACOSX_BUNDLE_COPYRIGHT "Copyright © %Y Cemu Project")
```

Comment:
I think it'd be better to replace this dynamic year with a static one, as it makes bit-by-bit build reproducibility a bit harder. Also, copyright applies from when you created (or significantly updated) a certain work, not when it was transformed in binary form.

Reply:
Yeah, that makes sense. Fixed.